docs/item.rst

```diff
@@ -0,0 +1,93 @@
+item
+====
+
+.. dfhack-tool::
+    :summary: Perform bulk operations on items based on various properties.
+    :tags: fort productivity items
+
+Filter items in you fort by various properties (e.g., item type, material,
+wear-level, quality, ...), and perform the bulk operations forbid, dump, melt,
+and their inverses. By default, the tool does not consider artifacts. Outputs
+the number of items that matched the filters and were modified.
+
+Usage
+-----
+
+``item [ help | count | [un]forbid | [un]dump | [un]melt ] <filter option>``
```

Comment:
```suggestion
``item [ count | [un]forbid | [un]dump | [un]melt ] <filter options>``
```

Reply:
supporting `help` as a command is fine, but the canonical way to get help is `help item`, so we don't want to promote alternatives.